README.md

```diff
@@ -27,18 +27,13 @@ $ yarn add @topcli/prompts
 You can locally run `node ./demo.js`
 
 ```js
-import { prompt, confirm, select } from '@topcli/prompts';
+import { question, confirm, select } from "./index.js";
```

Comment:
This code snippet represents a basic usage example using the package as third party library, not locally 
```suggestion
import { question, confirm, select } from "@topcli/prompts";
```